hdat/archive.py

```diff
@@ -1,10 +1,8 @@
 import os
-import json
-import copy
 import pickle
-from collections import namedtuple
+from collections.__init__ import namedtuple
```

Comment:
Why do we need `collections.__init__`?

Reply:
Not too sure; this changed automatically when I used PyCharm to refactor it.

Reply:
I would revert it.  This seems weird and unnecessary to me.